src/embed/world-renderer.js

```diff
@@ -168,13 +168,37 @@ WorldRenderer.prototype.submitFrame = function() {
     this.vrDisplay.submitFrame();
   }
 };
-
+WorldRenderer.prototype.dispose = function() {
+    var eyeLeft = this.scene.getObjectByName('eyeLeft'),
+        eyeRight = this.scene.getObjectByName('eyeRight');
+    if(!eyeLeft || !eyeRight) return
+    if(eyeLeft.material.map ) eyeLeft.material.map.dispose();
+       eyeLeft.geometry.dispose();
+
+    if(eyeRight.material.map ) eyeRight.material.map.dispose();
+       eyeRight.geometry.dispose();
+}
 WorldRenderer.prototype.destroy = function() {
```

Comment:
There is a lot of code duplication between dispose() and destroy(). Please factor the shared functionality out.